packages/registry/src/mainnet/chainflipPairs.ts

```diff
@@ -0,0 +1,61 @@
+import { Chain, Token } from "@/types";
+import { AssetHub, Ethereum, Polkadot } from "./chains";
+import { EthereumTokens, PolkadotTokens } from "./tokens";
+
+export type ChainflipSwapPair = [Token, Token];
+export type ChainflipSwapRoute = {
+  from: Chain;
+  to: Chain;
+  pairs: ChainflipSwapPair[];
+};
+
+export const chainflipSwapPairs: ChainflipSwapRoute[] = [
+  {
+    from: Ethereum,
+    to: AssetHub,
+    pairs: [
+      [EthereumTokens.ETH, PolkadotTokens.DOT],
+      [EthereumTokens.ETH, PolkadotTokens.USDC],
+      [EthereumTokens.ETH, PolkadotTokens.USDT],
+      [EthereumTokens.USDC, PolkadotTokens.DOT],
+      [EthereumTokens.USDC, PolkadotTokens.USDC],
+      [EthereumTokens.USDC, PolkadotTokens.USDT],
+      [EthereumTokens.USDT, PolkadotTokens.DOT],
+      [EthereumTokens.USDT, PolkadotTokens.USDC],
+      [EthereumTokens.USDT, PolkadotTokens.USDT],
+    ],
```

Comment:
We could make these route entries a map to avoid duplication and reduce search complexity. In pseudo-code:

```suggestion
    pairs: {
      'EthereumTokens.ETH': [PolkadotTokens.DOT, PolkadotTokens.USDC, PolkadotTokens.USD],
      'EthereumTokens.USDC':  [PolkadotTokens.DOT, PolkadotTokens.USDC, PolkadotTokens.USDT, ,
      'EthereumTokens.USDT': [PolkadotTokens.DOT, PolkadotTokens.USDC, PolkadotTokens.USDT],
    }
```